ASM/c/get_items.c

```diff
@@ -636,6 +659,19 @@ int16_t get_override_drop_id(int16_t dropId) {
         }
     }
 
+    // Chus in logic drops, convert bomb drop to bombchu drop under certain circumstances
```

Comment:
Adding this more as a reminder in myself to take the "Chus in logic" out of this comment